pkgs/development/python-modules/keyring-pass/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, gnupg
+, keyring
+, pass
+, poetry-core
+, pythonOlder
+}:
+buildPythonPackage rec {
+  pname = "keyring-pass";
+  version = "0.9.2";
+  disabled = pythonOlder "3.6";
+
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "nazarewk";
+    repo = "keyring_pass";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Sf7eDOB3prH2s6BzdBtxewSweC0ibLXVxNHBJRRaJe4=";
+  };
+
+  postPatch = ''
+    substituteInPlace keyring_pass/__init__.py \
+      --replace 'pass_binary = "pass"' 'pass_binary = "${pass}/bin/pass"'
```

Comment:
```suggestion
      --replace-fail 'pass_binary = "pass"' 'pass_binary = "${lib.getExe pass}"'
```

Reply:
nice, didn't know about this one

Reply:
I was right not to know about it, it wasn't there when I started the PR, had to rebase onto `nixos-unstable` ;